src/plugins/discover/server/sample_data/register_sample_data.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { i18n } from '@kbn/i18n';
+import type { SampleDataRegistrySetup } from '../../../home/server';
+import { APP_ICON } from '../../common';
+
+// TODO: update
+const getSampleSearchPath = (objId: string) => `/app/discover#/view/${encodeURIComponent(objId)}`;
```

Comment:
@kertal to check it

Reply:
Linking to https://github.com/elastic/kibana/issues/131328